Client/Frontend/Sync/BraveCore/Bookmarks/Bookmarkv2.swift

```diff
@@ -10,20 +10,22 @@ import BraveShared
 import CoreData
 import Shared
 
-private let log = Logger.browserLogger
-
 // A Lightweight wrapper around BraveCore bookmarks
 // with the same layout/interface as `Bookmark (from CoreData)`
 class Bookmarkv2: WebsitePresentable {
```

Comment:
This class seems a bit useless now, no?

It seems to be a very small wrapper around `bookmarkNode`. Why not just extend `BookmarkNode` with a few helper methods that you need in this class and use it directly in place of `Bookmarkv2`

Reply:
Bookmarkv2 is removed and BookmarkNode extension is created with helper methods and usage integration

Fix at https://github.com/brave/brave-ios/pull/4024/commits/082e60c0456a66026f2e69be1947b8e7f9db98ad